Dnn.AdminExperience/Dnn.PersonaBar.Extensions/Components/Pages/BulkPagesController.cs

```diff
@@ -111,6 +117,60 @@ protected override Func<IBulkPagesController> GetFactory()
             return () => new BulkPagesController();
         }
 
+        /// <summary>
+        /// Checks whether there is a duplicate tab exists.
+        /// </summary>
+        /// <param name="pageIndex">Page index to check the duplicate for.</param>
+        /// <param name="pages">Page list to check the duplicates from.</param>
+        /// <returns>True in case there is a tab with the same name and path and with the index of greater than the first occurence; false otherwise.</returns>
+        private static bool DuplicateExists(int pageIndex, string[] pages)
+        {
+            var pageName = pages[pageIndex];
+            var pageDirectory = GetPageDirectory(pageIndex, pages);
+
+            for (var index = 0; index < pageIndex; index++)
+            {
+                var loopedDirectory = GetPageDirectory(index, pages);
+                if (loopedDirectory != pageDirectory)
+                {
+                    continue;
+                }
+                else if (pageName == pages[index])
```

Comment:
This should probably be a case-insensitive check

Reply:
Fixed.